zerver/lib/push_notifications.py

```diff
@@ -396,15 +396,18 @@ def send_notifications_to_bouncer(
     apns_payload: Dict[str, Any],
     gcm_payload: Dict[str, Any],
     gcm_options: Dict[str, Any],
-) -> None:
+) -> Tuple[int, int]:
     post_data = {
         "user_id": user_profile_id,
         "apns_payload": apns_payload,
         "gcm_payload": gcm_payload,
         "gcm_options": gcm_options,
     }
     # Calls zilencer.views.remote_server_notify_push
-    send_json_to_push_bouncer("POST", "push/notify", post_data)
+    response_data = cast(
```

Comment:
Hmm, can we use one of the alternatives documented here?
https://zulip.readthedocs.io/en/latest/testing/mypy.html#avoid-cast

Reply:
E.g. `android_num = int(response['android_num'])` and then return the tuple of numbers of this form?